ProcessMaker/Events/TemplateUpdated.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace ProcessMaker\Events;
+
+use Carbon\Carbon;
+use Illuminate\Foundation\Events\Dispatchable;
+use ProcessMaker\Contracts\SecurityLogEventInterface;
+use ProcessMaker\Models\ProcessTemplates;
+use ProcessMaker\Traits\FormatSecurityLogChanges;
+
+class TemplateUpdated implements SecurityLogEventInterface
+{
+    use Dispatchable, FormatSecurityLogChanges;
+
+    private $changes;
+    private $process;
+    private $processType;
+
+    /**
+     * Create a new event instance.
+     *
+     * @return void
+     */
+    public function __construct(array $changes, $process = null, $processType)
+    {   
+        $this->changes = $changes;
+        $this->process = $process;
+        $this->processType = $processType;
+    }
+
+    /**
+     * Get specific data related to the event
+     *
+     * @return array
+     */
+    public function getData(): array
+    {
+        if ($this->processType == "Update Template Process") {
+            //for process Changes
+            $old_template_data = array_intersect_key($this->process->getOriginal(), array_flip(['updated_at']));
+            $old_template_data['updated_at'] = date('Y-m-d H:i:s', strtotime($old_template_data['updated_at']));
+
+            return [
+                'name' => [
+                    'label' => $this->processType
+                ],
+                '+ Process/Template_Last_Modified' => Carbon::now()
```

Comment:
Rename '+ Process/Template_Last_Modified' to: updated_at
Please use the value defined in process_templates.updated_at